src/_blog/cryptotls.md

```diff
@@ -0,0 +1,60 @@
+---
+tags:
+- quic
+- go
+- standard library
+- crypto/tls
+title: A QUIC API for crypto/tls
+description:
+date: 2023-08-15
+permalink: "/2023-08-15-quic-crypto-tls/"
+translationKey: ''
+header_image: /quic-crypto-tls.png
+author: Marten Seemann
+---
+
+# A QUIC API for crypto/tls
+
+You might have heard us talk about this before: QUIC has become the most important transport in libp2p. For example, in the IPFS network, QUIC connections account for a whopping 80-90% of the connections. go-libp2p uses [quic-go](https://github.com/quic-go/quic-go), a QUIC implementation written in pure Go. quic-go doesn’t only power QUIC support in go-libp2p, but also in [Caddy](https://caddyserver.com/), a full-featured webserver (for its HTTP/3 support), [Adguard](https://github.com/AdguardTeam/AdGuardHome) (for providing DNS over QUIC), and in the synchronization tool [syncthing](https://github.com/syncthing/syncthing/), and [many other projects](https://github.com/quic-go/quic-go#projects-using-quic-go).
```

Comment:
```suggestion
You might have [heard us talk about this before](https://www.youtube.com/watch?v=aDHymXQJ4bs): QUIC is beginning to surpass TCP as the most important non-browser transport in libp2p. For example, in the IPFS network, QUIC connections account for a whopping 80-90% of the connections. go-libp2p uses [quic-go](https://github.com/quic-go/quic-go), a QUIC implementation written in pure Go. quic-go doesn’t only power QUIC support in go-libp2p, but also in [Caddy](https://caddyserver.com/), a full-featured webserver (for its HTTP/3 support), [Adguard](https://github.com/AdguardTeam/AdGuardHome) (for providing DNS over QUIC), and in the synchronization tool [syncthing](https://github.com/syncthing/syncthing/), and [many other projects](https://github.com/quic-go/quic-go#projects-using-quic-go).
```

Is the first sentence in reference to https://www.youtube.com/watch?v=aDHymXQJ4bs ? 

> QUIC has become the most important transport in libp2p

to "QUIC is beginning to surpass TCP as the most important non-browser transport in libp2p"
wdyt? While it's true for IPFS/Filecoin & go-libp2p, there are other projects and implementations that don't use QUIC yet (Ethereum, etc.)

> "whopping 80-90% of the connections"

link?

Reply:
Link would be the swarm metrics, once we have our Grafana instance up and running. 80-90% is way more than "beginning to surpass" though.